src/pages/accounts/ProfilePage.tsx

```diff
@@ -40,8 +40,9 @@ const ProfilePage = (): JSX.Element => {
       const data = await updateUser(token, profile)
       setProfileContext(data)
       navigate('/account')
+      addErrorCallback({ body: t('text.userUpdated'), variant: 'success' })
```

Comment:
Let's wait for the follow up refactor to create the success or this is confusing